client/my-sites/stats/hooks/use-site-complusory-plan-selection-qualified-check.ts

```diff
@@ -9,17 +9,11 @@ export default function useSiteComplusoryPlanSelectionQualifiedCheck( siteId: nu
 		getSiteOption( state, siteId, 'created_at' )
 	) as string;
 	const { isPending, data: usageInfo } = usePlanUsageQuery( siteId );
-	// `recent_usages` is an array of the most recent 3 months usage data, and `current_usage` is the current month usage data.
-	// Note: two of the highest days every month in the last 3 months are excluded before calculating the monthly views.
-	const recentMonthlyViews = Math.max(
-		usageInfo?.recent_usages[ 0 ]?.views_count ?? 0,
-		usageInfo?.recent_usages[ 1 ]?.views_count ?? 0,
-		usageInfo?.recent_usages[ 2 ]?.views_count ?? 0,
-		usageInfo?.current_usage?.views_count ?? 0
-	);
+
 	const isNewSite =
 		siteCreatedTimeStamp && new Date( siteCreatedTimeStamp ) > new Date( '2024-01-31' ); // Targeting new sites
-	const isExistingSampleSite = recentMonthlyViews > MIN_MONTHLY_VIEWS_TO_APPLY_PAYWALL; // Targeting all existing sites with views higher than 1000/mth.
+	const isExistingSampleSite =
+		usageInfo?.billableMonthlyViews ?? 0 > MIN_MONTHLY_VIEWS_TO_APPLY_PAYWALL; // Targeting all existing sites with views higher than 1000/mth.
```

Comment:
This looks like a typo with `0`.

Reply:
Oh this is intended - mostly for the sake of typing

Reply:
Thanks for catching this! Addressed in f74dbefef06